### DISCUSSION TITLE
Application for cooperation from Goodrain open source community

### DISCUSSION THREAD

**Discussion by zzzhangqi** [apache/skywalking#8866] [General]
Hello,I'm from the open source community [Goodrain](https://github.com/goodrain).The open source project we operate is [Rainbond](https://github.com/goodrain/rainbond).Our [website](https://www.rainbond.com/).

We have many users using skywalking.So we provide a choice for these users through Rainbond through our own application market mechanism.One-click deployment of Skywalking available for production.This kind of deployment is very popular with community users.

Therefore, a possibility of cooperation is hereby put forward,We are willing to submit an one-click deployment Skywalking document to your website or blog,Help more users who need both cloud native application management platform and Skywalking to understand this possibility and its convenience.

You can watch the way we worked with [Nacos](https://nacos.io/zh-cn/blog/use-nacos-with-rainbond.html) and [Apollo](https://www.apolloconfig.com/#/zh/deployment/third-party-tool-rainbond) in the past.

Looking forward to the reply from the operators of the Skywalking community!😚

**Comment by wu-sheng** [comment]
I am not sure whether you are requesting this, I remember you used to have a pull request on the website repo, I had given comments, https://github.com/apache/skywalking-website/pull/410#issuecomment-1072235016 and  https://github.com/apache/skywalking-website/pull/410#issuecomment-1072247444

What is the difference comparing to that one?

**Reply by zzzhangqi** [reply]
Last time,document focuses on how skywalking is used on Rainbond.It may introduce us too much and apply to our users.

This paper mainly introduces the one-click installation of skywalking through rainbond open source application market.I think this way allows skywalking users to add a new installation method.

**Reply by zzzhangqi** [reply]
It's like the way we worked with [Nacos](https://nacos.io/zh-cn/blog/use-nacos-with-rainbond.html) and [Apollo](https://www.apolloconfig.com/#/zh/deployment/third-party-tool-rainbond) in the past.

**Reply by wu-sheng** [reply]
I think my preference is the same as last time. The official document is only including codes and features implemented by the project.

**Reply by zzzhangqi** [reply]
OK, thank you very much for your reply.
